{"name": "test_negative_login_with_Wrong_User_Information_2___asd_gmail_com____as__________________", "status": "broken", "statusDetails": {"message": "TypeError: TestLoginFunctionality.test_negative_login_with_Wrong_User_Information() takes 3 positional arguments but 7 were given", "trace": "self = <tests.test_loginTest.TestLoginFunctionality testMethod=test_negative_login_with_Wrong_User_Information_2___asd_gmail_com____as__________________>\n\n    @wraps(func)\n    def wrapper(self):\n>       return func(self, *args, **kwargs)\nE       TypeError: TestLoginFunctionality.test_negative_login_with_Wrong_User_Information() takes 3 positional arguments but 7 were given\n\nvenv\\Lib\\site-packages\\ddt.py:220: TypeError"}, "start": 1677149215534, "stop": 1677149215534, "uuid": "90f6d4d1-228f-45c7-bbda-dbd976e91210", "historyId": "15a755b3cc8ec3a7e04f6de7414684dc", "testCaseId": "57527433f2092ea094cd5384a3c5d5a3", "fullName": "tests.test_loginTest.TestLoginFunctionality#test_negative_login_with_Wrong_User_Information_2___asd_gmail_com____as__________________", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_loginTest"}, {"name": "subSuite", "value": "TestLoginFunctionality"}, {"name": "host", "value": "DESKTOP-75D171T"}, {"name": "thread", "value": "13816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_loginTest"}]}